### PR TITLE
makes wheelchairs able to pass through turnstiles (maybe((idontactuallyknowhowtocode(((help)))

### LINKED ISSUE #12903: Paraplegic people are unable to properly utilize turnstiles
### Severity

Inconvenience

### Short Description

Turnstiles do not have wheelchair access, as a result paraplegic people need to leave their chair to get access.  Doing this generally results in them being stepped on, skipped in line, and having their wheelchair stolen.

### Reproduction

1. Try to enter a turnstile with a wheelchair.
2. Get denied 

### Round ID

N/A

### Test Merges

_No response_

### BYOND Username

Cark

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by UselessTheremin**
if this doesnt work its a learning experience
fixes #12903 
:cl:  
bugfix: wheelchairs can now use turnstiles  
/:cl:

### CODE REVIEW

**Comment by Altoids1** [comment]
I remember the exact round where you realized that this problem existed. 😁 

It'd be ~~more ableist and funnier~~ better if there were a random chance of you being unable to fit through, or it was a progress-bar kinda effort to fit through the space.

File: code/game/machinery/doors/ministile.dm [hunk 1]
```diff
@@ -31,7 +31,7 @@
 	. = ..()
 	if(istype(mover) && (mover.pass_flags & PASSGLASS))
 		return TRUE
-	if(istype(mover, /mob/living/simple_animal/bot))
+	if(istype(mover, /mob/living/simple_animal/bot) || istype(/obj/vehicle/ridden/wheelchair))
```
Comment (SomeguyManperson) [inline code/game/machinery/doors/ministile.dm]:
```suggestion
	if(istype(mover, /mob/living/simple_animal/bot) || istype(mover, /obj/vehicle/ridden/wheelchair))
```

**Comment by maxion12345** [comment]
how the duck are you gonna fit a wheelchair through a turnstile

**Comment by UselessTheremin** [comment]
> how the duck are you gonna fit a wheelchair through a turnstile

i dunno but how the fuck you gonna get your access changed as a paraplegic if you cant fit through a turnstyle

**Comment by JamieD1** [comment]
Does it work?

File: code/game/machinery/doors/ministile.dm [hunk 2]
```diff
@@ -31,7 +31,7 @@
 	. = ..()
 	if(istype(mover) && (mover.pass_flags & PASSGLASS))
 		return TRUE
-	if(istype(mover, /mob/living/simple_animal/bot))
+	if(istype(mover, /mob/living/simple_animal/bot) || istype(mover, /obj/vehicle/ridden/wheelchair))
```
Comment (adamsong) [inline code/game/machinery/doors/ministile.dm]: Will this not make turnstiles AA for anyone in a wheelchair?

**Review by adamsong — DISMISSED** [review]

File: code/game/machinery/doors/ministile.dm (hunk 2 above)
Comment (neeshacark) [reply code/game/machinery/doors/ministile.dm]: i for one think this is an excellent change

Comment (ToasterBiome) [reply code/game/machinery/doors/ministile.dm]: This does.

**Comment by bruhlookatthisdood** [comment]
day 31, i am still stuck outside of the brig with my legs crippled
the ai refuses to open and the clown is coming with their horn.

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**